# We need <unistd.h>.
# This patch has been submitted to the community.
--- a/gold/timer.cc	2014-10-14 00:32:04.000000000 -0700
+++ b/gold/timer.cc	2015-08-04 11:59:42.929031188 -0700
@@ -28,6 +28,8 @@
 #include <sys/times.h>
 #endif
 
+#include <unistd.h>
+
 #include "libiberty.h"
 
 #include "timer.h"
